templates/openstack/managed.html

```diff
@@ -30,7 +30,7 @@ <h2>Cut your infrastructure costs with Managed OpenStack</h2>
           <p>Offload from your operations team and accelerate your time-to-market with Canonical’s Managed OpenStack.</p>
         </div>
         <div class="p-cta-block">
-          <a href="/openstack/contact-us"
+          <a href="/openstack/managed#get-in-touch"
```

Comment:
```suggestion
          <a href="/openstack/contact-us"
```
It should link to the static contact form as a JS fallback. If user has JS enabled, the cta button will trigger modal to open and attach the `#get-in-touch` hash to the URL

I left a suggestion on the copy doc for this change as well